### PR TITLE
Fix to a bug (Not ready)

### PR DESCRIPTION
**Pull request by Begad666**
Fixes a bug where if 2 links are created in same time only 1 will be used

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -42,41 +42,38 @@ async function connect() {
...
 app.post('/gen/rr', bodyParser, (req, res) => {
-    console.log("Got a request for creating RR!")
     const params = req.body
     
     if (!params.url || !params.title){
         res.status(400).send({error: "bad request"})
     }
 
-    info.url = {title: params.title , description: params.description , ImgUrl: params.ImgUrl}
+    info[url] = {title: params.title , description: params.description , ImgUrl: params.ImgUrl}
```
Comment (NihalNavath) [inline index.js:76]: url is not defined, (on line 76) either should be params.url or define url

Comment (Begad666) [reply index.js]: lol forgot to change it

**Review by Begad666 — COMMENTED** [review]
k done

**Comment by NihalNavath** [comment]
I have incoprated the changes in pr #2, thank you.

**Pull request closed without merging**